### PR TITLE
Blog app - controllers

### PR DESCRIPTION
**Pull request by katarighe**
In this section, I have completed the following activities: 

1) **Did NOT** use rails generate scaffold command. The point of this project is to show you how to set up controllers by myself.

2) **Ensured** that my app handles the URLs used in the wireframes:

3) For each URL, **I created** the following items:
- A route.
- An action in the correct controller.
- A view file.

4) **View files** should include only placeholders written in plain HTML, _e.g: Here is a list of posts for a given user_. 

5) **Ensured** that I added code to handle only the 4 required URLs.

### CODE REVIEW

File: app/controllers/users_controller.rb [hunk 1]
```diff
@@ -0,0 +1,5 @@
+class PostsController < ApplicationController
+  def index; end
+
+  def show; end
+end
```
Comment (thecodechaser) [inline app/controllers/users_controller.rb]: - [x] The class name for users controller should be `UsersController`. Please make sure to change the class name to `UsersController`.

File: config/routes.rb [hunk 2]
```diff
@@ -1,10 +1,6 @@
...
-  # Can be used by load balancers and uptime monitors to verify that the app is live.
-  get "up" => "rails/health#show", as: :rails_health_check
-
-  # Defines the root path route ("/")
-  # root "posts#index"
-end
+  root "users#index"
+  resources :users, only: [:index, show] do
+    resources :posts, only: [:index, show]
+  end
```
Comment (thecodechaser) [inline config/routes.rb]: - [x] You missed adding the colon `:` to show property. Please make sure to fix this so the routes work as expected.
  - You can consider using the below code.
  
```ruby
resources :users, only: [:index, :show] do
    resources :posts, only: [:index, :show]
  end

**Review by thecodechaser — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED ♻️ 

## Hi @katarighe👋, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## To Highlight 🎉
- Good job with controllers. ✔️
- Gitflow is followed. :heavy_check_mark:
- Professional README file. :heavy_check_mark:
- No linter errors. :heavy_check_mark:
- PR has a good title and summary. ✔️



## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by anagudelogu — APPROVED** [review]
# STATUS: APPROVED! 🍾 🎆 🥇 

Hi @katarighe, 
I'm Andres.

You did a great job working on this milestone, your project is complete! 🚀 
There is nothing else to say other than... it's time to merge it 🤖  
Congratulations! 🎉 



## Highlights
✅ Great use of Git flow. 💯 
✅ Your project is documented professionally. 📜 
✅ Great UI design. 💻 
✅ Descriptive PR title, summary and commit messages. 💬 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**